fog/view/server/src/block_tracker.rs

```diff
@@ -154,6 +154,25 @@ where
                     continue;
                 }
 
+                // Skip the the below checks when the next block index is less than the shard's
+                // start block. This prevents prior missed blocks from from impeding the shard's
```

Comment:
:upside_down_face: 
```suggestion
                // start block. This prevents prior missed blocks from impeding the shard's
```

Reply:
Removed in later commits. Marking as resolved.